appveyor.yml

```diff
@@ -1,22 +1,20 @@
 # Fix line endings in Windows. (runs before repo cloning)
 init:
   - git config --global core.autocrlf input
-  - git config --global user.email "you@example.com"
```

Comment:
I never thought of that these `init` steps is test suite setup scripts.

Maybe we shall make more specific email/name to document that they are for test purpose.

It comes to me like
```bash
git config --global user.email "hexo-deployer-git-test@example.com"
- git config --global user.name "hexo-deployer-git Test"
```

Reply:
Ok, will use the specific email/name.